src/wp-includes/ms-blogs.php

```diff
@@ -666,7 +666,7 @@ function restore_current_blog() {
 				);
 			}
 
-			wp_cache_add_non_persistent_groups( array( 'counts', 'plugins', 'theme_json' ) );
```

Comment:
See above.

Reply:
It only removed `theme_json'` from each of these. https://core.trac.wordpress.org/changeset/55092